cpp/dolfinx/mesh/DistributedMeshTools.cpp

```diff
@@ -92,22 +80,23 @@ compute_entity_numbering(const Mesh& mesh, int d)
     std::map<std::vector<std::int64_t>, std::int32_t> entity_to_local_index;
 
     // Get number of vertices in this entity type
-    const mesh::CellType& ct = mesh.cell_type();
-    const mesh::CellType& et = mesh::cell_entity_type(ct, d);
+    const mesh::CellType& et = mesh::cell_entity_type(cell_type, d);
     const int num_entity_vertices = mesh::num_cell_vertices(et);
 
     // Make a mask listing all shared vertices (true if shared)
-    std::vector<bool> vertex_shared(mesh.num_entities(0), false);
+    auto map0 = topology.index_map(0);
+    assert(map0);
+    const int num_vertices = map0->size_local() + map0->num_ghosts();
+    std::vector<bool> vertex_shared(num_vertices, false);
     for (const auto& v : shared_vertices_local)
       vertex_shared[v.first] = true;
 
-    for (auto& e : mesh::MeshRange(mesh, d, mesh::MeshRangeType::ALL))
+    for (int e = 0; e < size; ++e)
```

Comment:
`size` is defined as `int32_t`, no need for `e` be `int` (i.e. 64-bit on some machines).

Reply:
I am planning to delete this code soon, anyway, as it is largely duplicated in `TopologyComputation`.